### PR TITLE
chore: do not use file upload dialog for community creation

### PR DESCRIPTION
**Pull request by anastasiyaig**
Depends on https://github.com/status-im/status-desktop/pull/13537
Fixes https://github.com/status-im/status-desktop/issues/13483

So community creation works on each host now.

Screenshot 2024-02-13 at 17 49 20

https://ci.status.im/job/status-desktop/job/e2e/job/manual/1396/ Linux 01
https://ci.status.im/job/status-desktop/job/e2e/job/manual/1397/ Linux 02
https://ci.status.im/job/status-desktop/job/e2e/job/manual/1398/ Linux 03
https://ci.status.im/job/status-desktop/job/e2e/job/manual/1399/ Linux 04
https://ci.status.im/job/status-desktop/job/e2e/job/manual/1400/ Linux 05

~Linux 03 is now excluded from DISABLED hosts on CI as well~ UPD: i realised there are other places in the app where we use the file dialog, so i will handle them and then enable the host back

All community tests work now https://ci.status.im/job/status-desktop/job/e2e/job/manual/1403/ (i excluded one)
Edit community test is back to critical path

### CODE REVIEW

**Comment by anastasiyaig** [comment]
@Valentina1133 i skipped 1 test there , it has to be removed to be honest. Also, there is still more work on that to be done (fix the same thing in profile , we use file dialogs in onboarding and settings)
I was super tired to make it work today.i suggest we merge that and i will fix the rest separately, wdyt?

File: gui/components/community/create_community_popups.py [hunk 1]
```diff
@@ -149,12 +160,16 @@ def outro(self, value: str):
...
+        self.set_banner_without_file_upload_dialog(kwargs['banner']['fp'])
+        PictureEditPopup().make_picture(None, None)
         for key in list(kwargs):
             if key in ['intro', 'outro'] and self._next_button.is_visible:
                 self._next_button.click()
             setattr(self, key, kwargs.get(key))
         self._create_community_button.click()
         self.wait_until_hidden()
-        return CommunityScreen().wait_until_appears()
+        return CommunityScreen().wait_until_appears()
```
Comment (Valentina1133) [inline gui/components/community/create_community_popups.py:175]: Extra row needed

**Review by Valentina1133 — APPROVED** [review]

File: gui/components/community/create_community_popups.py (hunk 1 above)
Comment (anastasiyaig) [reply gui/components/community/create_community_popups.py]: damn, fixed :)

**Comment by anastasiyaig** [comment]
Follow up task https://github.com/status-im/desktop-qa-automation/issues/513

**Pull request merged**